### ISSUE TITLE
Open source of the RoboData

### ISSUE THREAD

**Issue by jiafei1224** [RoboUniview/RoboMM#2]
Hi, 

Great work! I would like to know when can i get full access to the RoboData, and if it is possible to get it soon?

Regards,
Jiafei

**Comment by fengxiuyaun** [comment]
Thank you for your interest in RoboData! We are currently in the process of uploading the data, and due to its large volume, it might take until the end of this month. We are also organizing the data processing code, so you can also regenerate it from the raw data.

**Comment by aopolin-lv** [comment]
> Thank you for your interest in RoboData! We are currently in the process of uploading the data, and due to its large volume, it might take until the end of this month. We are also organizing the data processing code, so you can also regenerate it from the raw data.

Sounds awesome! Looking forward to the data and scripts.

**Comment by fengxiuyaun** [comment]
Done！ https://huggingface.co/datasets/liufanfanlff/RoboData

**Comment by Heron0625** [comment]
> Done！ https://huggingface.co/datasets/liufanfanlff/RoboData

Thank you for open sourcing your data. I noticed that your article says that RoboData is made up of CALVIN , MetaWorld , LIBERO , Robomimic , RoboCasa , ManiSkill2 , RoboCAS , RLBench and Colosseum . The first five are included in your currently publicly available data. Can I ask if the dataset that is now publicly available is the final version?

**Comment by fengxiuyaun** [comment]
Yes. We are currently using 9 datasets to train our model. Due to the large volume of data, the training time will be a bit longer. Please stay tuned.

**Comment by Heron0625** [comment]
> Yes. We are currently using 9 datasets to train our model. Due to the large volume of data, the training time will be a bit longer. Please stay tuned.

I don't seem to have phrased my question clearly. What I meant was that the paper mentions nine datasets, but the above link only contains five. Are there any subsequent plans to make the remaining few standardised datasets publicly available?

**Comment by fengxiuyaun** [comment]
> > Yes. We are currently using 9 datasets to train our model. Due to the large volume of data, the training time will be a bit longer. Please stay tuned.
> 
> I don't seem to have phrased my question clearly. What I meant was that the paper mentions nine datasets, but the above link only contains five. Are there any subsequent plans to make the remaining few standardised datasets publicly available?

Yes, Other datasets are also being prepared and will be uploaded later.
